### ISSUE TITLE
Curious..

### ISSUE THREAD

**Issue by 0xdevalias** [OlegIlyenko/hacking-scala-blog#1]
Heya,

I'm curious, are you just manually tracking your posts to tumblr here, or do you have a nice github/markdown -> tumblr build integration setup?

**Comment by OlegIlyenko** [comment]
no, I'm doing it manually. just as a backup and to have some versioning. I started it as I was migrating from posterous and then continued after i finished migration to tumblr

**Comment by 0xdevalias** [comment]
Fair enough :)

I'm sort of halfway between moving between Tumblr and something like octopress, but sort of like the idea of keeping it mirrored/shareable on Tumblr.

—
